docs/class-definitions.md

```diff
@@ -355,7 +355,7 @@ space.locations.unsubscribe('update');
 Get location for self.
 
 ```ts
-type getSelf = () => Promise<Location | undefined>;
+type getSelf = () => Promise<Location>;
```

Comment:
`Location`, as defined in this file, already includes `null` in it's type definition